gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-pages-list/api-documentation-v4-pages-list.component.html

```diff
@@ -25,11 +25,11 @@
     <ng-container matColumnDef="name">
       <th mat-header-cell *matHeaderCellDef>Name</th>
       <td mat-cell *matCellDef="let page">
-        <div class="documentation-pages-list__table__name clickable" *ngIf="page.type === 'FOLDER'" (click)="onGoToFolder.emit(page.id)">
+        <div class="documentation-pages-list__table__name" *ngIf="page.type === 'FOLDER'" (click)="onGoToFolder.emit(page.id)">
           <mat-icon svgIcon="gio:folder"></mat-icon>
           <p>{{ page.name }}</p>
         </div>
-        <div class="documentation-pages-list__table__name" *ngIf="page.type !== 'FOLDER'">
+        <div class="documentation-pages-list__table__name" *ngIf="page.type !== 'FOLDER'" (click)="onGoToPage.emit(page.id)">
```

Comment:
I think this can also be added to the pencil action, but the event would be more `onEditPage`.